docs/core/tools/nuget-signed-package-verification.md

```diff
@@ -0,0 +1,44 @@
+---
+title: NuGet signed package verification
+description: NuGet uses root stores valid for code signing and timestamping to perform signed package verification.
+author: DTivel
+ms.date: 11/07/2022
+---
+# NuGet signed package verification
+
+NuGet package signatures are based on X.509 certificates, and a prequisite for signed package verification is a certificate root store that is valid for both code signing and timestamping.
+
+Starting with .NET 7, NuGet uses fallback certificate bundles included in the .NET SDK to verify signed packages where a suitable system root store is not available.  These bundles are sourced from the [Microsoft Trusted Root Program](https://aka.ms/RootCert) and contain the same code signing and timestamping certificates as the root store on Windows.
+
+Some NuGet commands (like `sign` and `verify`) always perform signed package verification.
+
+The following sections detail when implicit verification during restore operations is enabled by default, how to enable it, and what root stores are used.
+
+## Windows
+
+Verification is enabled by default during package restore operations.
+
+NuGet uses the default root store on Windows, which already supports general-purpose code signing and timestamping.  .NET SDK fallback certificate bundles are not used.
+
+## Linux
+
+Verification is disabled by default during package restore operations.  To opt in, set the environment variable `DOTNET_NUGET_SIGNATURE_VERIFICATION` to `true`.
```

Comment:
Shall we mention that "It is opt-in for .NET 7 Preview 7 and .NET 6 SDK (6.0.400)", but "will be enabled by default in .NET 7 SDK Linux builds, starting with RC1."?